meals/views.py

```diff
@@ -0,0 +1,188 @@
+from django.core.urlresolvers import reverse
+from meals.models import Meal, Eaten, MEAL_TYPE
+from meals.tables import MealsListTable, MealsListFilter, MealsUsersTable, MealsUsersFilter
+from app.mixins import TabsViewMixin
+from django_tables2 import SingleTableMixin
+from django_filters.views import FilterView
+from django.core import serializers
+from django.http import HttpResponse
+from user.models import User
+from rest_framework.permissions import AllowAny
+from rest_framework.views import APIView
+from applications.models import Application
+from app.views import TabsView
+from datetime import datetime
+from django.contrib import messages
+from django.shortcuts import redirect
+from django.http import Http404
+
+token = 'felix'
```

Comment:
Waht?

Reply:
Use secret instead? See settings.py. It's loaded from the environment (see `server.sh` in server)

Reply:
There's already an issue about that #46.